core/src/test/groovy/com/novoda/gradle/release/LibProjectFactory.groovy

```diff
@@ -1,53 +0,0 @@
-package com.novoda.gradle.release
-
-import org.gradle.api.Project
-import org.gradle.testfixtures.ProjectBuilder
-
-final class LibProjectFactory {
```

Comment:
I know a guy who created some pretty slick integration tests for Gradle plugins, you should ask him to look into this 👀